service/gradle/deploy.gradle

```diff
@@ -0,0 +1,24 @@
+import java.time.ZonedDateTime
+
+// Deploy config
+jib {
+  from {
+    // see https://github.com/broadinstitute/dsp-appsec-blessed-images/tree/main/jre
+    // Google's distroless images are openjdk, this is the simplest with adoptopenjdk
+    image = "us.gcr.io/broad-dsp-gcr-public/base/jre:11-debian"
+  }
+  extraDirectories {
+    paths = [file(jibExtraDirectory)]
+  }
+  container {
+    filesModificationTime = ZonedDateTime.now().toString() // to prevent ui caching
+    jvmFlags = [
+            "-agentpath:" + cloudProfilerLocation + "/profiler_java_agent.so=" +
```

Comment:
That variable is defined in the profiler part. I think it would be better if we avoided references across the leaf parts.

So maybe combine or set in the top-level file?